cpp/Platform.Setters/Setter[TResult].h

```diff
@@ -1,10 +1,10 @@
-﻿namespace Platform::Setters
-{
-    template <typename ...> class Setter;
-    template <typename TResult> class Setter<TResult> : public Setter<TResult, bool>
-    {
-        public: Setter(TResult defaultValue) : Setter<TResult, bool>(true, false, defaultValue) { }
-
-        public: Setter() : Setter<TResult, bool>(true, false) { }
-    };
-}
+﻿namespace Platform::Setters
+{
+    template <typename ...> class Setter;
+    template <typename TResult> class Setter<TResult> : public Setter<TResult, bool>
+    {
+        public: Setter(TResult defaultValue) : Setter<TResult, bool>(true, false, defaultValue) { }

```

Comment:
`std::move(defaultValue)`

Reply:
Do we really need to move integral types?